Fix Dumpling capitalization: product name should be capitalized (#23042)
pingcap/docs: dm/dm-faq.md

You can avoid this issue in advance by configuring in the following ways:

> 1. Increase the value of `expire_logs_days` in the upstream MySQL database to avoid wrongly purging needed binlog files before the full migration task completes. If the data volume is large, we recommend that you use Dumpling and TiDB Lightning at the same time to speed up the task.

Reviewer: ```suggestion
1. Increase the value of `expire_logs_days` in the upstream MySQL database to avoid wrongly purging needed binlog files before the full migration task completes. If the data volume is large, it is recommended to use Dumpling and TiDB Lightning at the same time to speed up the task.
```
